src/main/kotlin/net/minecrell/pluginyml/paper/PaperPluginDescription.kt

```diff
@@ -53,10 +55,10 @@ class PaperPluginDescription(project: Project) : PluginDescription() {
     @Input @Optional var contributors: List<String>? = null
     @Input @Optional var website: String? = null
     @Input @Optional var prefix: String? = null
-    @Input @Optional @JsonProperty("defaultPerm") var defaultPermission: Permission.Default? = null
```

Comment:
I believe this one is still called defaultPerm so the property should stay. Or maybe you should nag someone from Paper about this so they use a consistent naming scheme :)

See https://github.com/PaperMC/Paper/blob/dc30ee591d52a55ac1f6dfce6e731a583cc21fad/patches/server/0013-Paper-Plugins.patch#L5161

Reply:
Uh. Is this the wrong code I linked? Where do you see `default-permission`?

Reply:
Strange, I must have been looking at the wrong revision. Could not find that at all.